.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
```

Comment:
Build implies a Go binary is being built here which doesn't seem accurate.

Reply:
Renamed to `goyek`